### PR TITLE
feat(coil-extension): update popup copy

### PR DESCRIPTION
**Pull request by sublimator**
Extension side of https://github.com/coilhq/coil/pull/2821

### CODE REVIEW

File: packages/coil-extension/src/popup/components/Unsubscribed.tsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ import { PopupProps } from '../types'
 import { StatusButton } from './StatusButton'
 import { StatusTypography } from './util/StatusTypography'
 
-const titleString = 'Get a subscription'
-const subheading1 = 'To use Coil you need an active subscription'
+const titleString = 'Get a membership'
```
Comment (sublimator) [inline packages/coil-extension/src/popup/components/Unsubscribed.tsx]: This seems kind of awkward. Is the plan to use membership on the site as well?

Comment (sublimator) [reply packages/coil-extension/src/popup/components/Unsubscribed.tsx]: image

^ This will become Membership too ?

Comment (traviscrist) [reply packages/coil-extension/src/popup/components/Unsubscribed.tsx]: The site will change over to membership once the new settings page Dacio is working on is merged in.

**Comment by traviscrist** [comment]
@fruehle does `Get a membership` sound ok to you, its for the extension state where you are signed into coil but don't have an active membership.

**Comment by fruehle** [comment]
@travis: We used ‘Become a member’ on the static pages for the CTA.

**Comment by sublimator** [comment]
sounds less awkward

**Comment by traviscrist** [comment]
@fruehle that sounds much better, @sublimator lets go with `Become a member`

**Pull request merged**